events/tempVoice.js

```diff
@@ -71,13 +71,16 @@ async function createMenu(client, categoryIds, channelId) {
     const botMessage = messages.find(message => message.author.id === client.user.id);
 
     if (botMessage) {
-        console.log('Bot message already exists in the create voice channel.');
-        return;
-    }
-
-    // If there are any messages, delete them
-    if (messages.size > 0) {
-        await channel.bulkDelete(messages);
+        console.log('Bot message already exists in the create voice channel. Deleting old messages...');
+
+        if (messages.size > 0) {
+            console.log("Deleting messages from previous create voice channel session...")
+            // await channel.bulkDelete(messages);
+            // Bulk delete doesn't work for messages older than 14 days
+            for (const message of messages.values()) {
+                await message.delete();
+            }
+        }
```

Comment:
Great solution! Small suggestion (feel free to ignore if I missed anything):

Adding error handling around individual deletions could help with debugging if any deletion fails:

```suggestion
        if (messages.size > 0) {
            console.log("Deleting messages from previous create voice channel session...")
            for (const message of messages.values()) {
                try {
                    await message.delete();
                } catch (err) {
                    console.error(`Failed to delete message ${message.id}:`, err);
                }
            }
        }
```

This allows cleanup to continue even if one deletion fails and logs what went wrong.

Thanks for the fix!